chapter4.Rmd

```diff
@@ -363,10 +357,10 @@ skills: 1
 key: 90ecc160d1ebf2f75bf53f9c3843fc1632bdd0a5
 ```
 
-In `factor_survey_vector` we have a factor with two levels: Male and Female. But how does R value these relatively to each other? In other words, who does R think is better, males or females?
+You might wonder what happens when you try to compare elements of a factor.  In `factor_survey_vector` you have a factor with two levels: `"Male"` and `"Female"`. But how does R value these relatively to each other? In other words, who does R think is greater, males or females?
```

Comment:
"In other words, which does R think should come first in sorted data, males or females?"

Reply:
"Greater" is used because the question is about using then greater than operator on factor values. So I think I'm happy with the sentence as it is.

Reply:
Agree with @gvwilson that this phrasing is fraught. Maybe rephrase as:

In other words, which level does R consider to be greater, `male` or `female`?

Therfore, we avoid using the word "value", and also by using inline code for `male` and `female` it is clear that we aren't referring to men and women in general.